src/objecttypes/utils/tests/keycloak.py

```diff
@@ -0,0 +1,89 @@
+from contextlib import contextmanager, nullcontext
+from unittest.mock import patch
+
+from django.apps import apps
+
+from pyquery import PyQuery as pq
+from requests import Session
+
+KEYCLOAK_BASE_URL = "http://localhost:8080/realms/test/protocol/openid-connect"
+
+
+def keycloak_login(
```

Comment:
Can't we reuse this from `mozilla-django-oidc-db`? https://github.com/maykinmedia/mozilla-django-oidc-db/blob/d0630df6a5dc9d8bf34dbed5a0be983973779ed6/tests/utils.py#L7

Reply:
This is reused from mozilla-django-oidc-db but with some slight alterations. Just as in open-forms.

![Screenshot from 2024-08-14 16-59-11](https://github.com/user-attachments/assets/4b8fda66-0faf-4394-b15d-f94fc8cdc961)